pwa/src/context/filters.ts

```diff
@@ -3,11 +3,13 @@ import * as React from "react";
 export type TComponentResultsLayout = "table" | "cards" | "layer";
 export type TLandingDisplayLayout = "layer" | "categories";
```

Comment:
I think a more logical name for this "categories" view would be "cards", like the other ones. Because it are cards after all. And I don't think "categories" is a type of layout.